draft-ietf-tls-ticketrequests.md

```diff
@@ -124,32 +146,73 @@ Clients MAY send this extension in ClientHello. It contains the following struct
 
 ~~~
 struct {
-    uint8 count;
-} TicketRequestContents;
-~~~
+    uint8 new_session_count;
+    uint8 resumption_count;
+} ClientTicketRequest;
 
-count
-: The number of tickets desired by the client.
+struct {
+    uint8 expected_count;
+} ServerTicketRequest;
```

Comment:
Should this really be called a ServerTicketRequest? It doesn't seem like it's making a request, but instead sharing info.

Reply:
It's the server's version of the "ticket_request" extension, so I think we should keep it!

Reply:
I think it's a bit misleading; it's not any sort of request. ServerTicketRequestHint or something maybe.

Reply:
Hmm, I disagree. Again, it's the server's "ticket_request" extension. I could live with `ServerTicketRequestHint.